daml-lf/transaction/src/main/scala/com/daml/lf/Error.scala

```diff
@@ -89,6 +89,15 @@ object Error {
       actual: TypeConName,
   ) extends Error
 
+  /** We tried to fetch / exercise a contract by interface, but
+    * the contract does not implement this interface.
+    */
+  final case class ContractDoesntImplementInterface(
```

Comment:
bikeshed: Should it be `ContractDoesNotImplementInterface`?

Reply:
done!